### PR TITLE
Balance changesfor 3.14.3

### PR DESCRIPTION
**Pull request by SebastianTanoni**

### CODE REVIEW

**Comment by SebastianTanoni** [comment]
#2816 #2817 #2819 #2820

**Comment by SebastianTanoni** [comment]
Changelog

-Storm cloud smaller
-Scarlet Templar now have normal Holy Light
-Building orgrimmar now costs 200 gold and doesnt reward any
-Quest shared vision from turn 14 to turn 16
-Made creeping buildings neutral passive so you do not need to destroy them to get the quest
-Removed some trees that were blocking common pathing areas or making them too narrow. (quelthalas, ashenvale mostly)
-added blockers instead of trees next to barrow dens,
removing those blockers when malfurion is unlocked

File: src/WarcraftLegacies.Source/Quests/Warsong/QuestOrgrimmar.cs [hunk 1]
```diff
@@ -32,12 +31,11 @@ public sealed class QuestOrgrimmar : QuestData
     public override string RewardFlavour => "The city of Orgrimmar was finally constructed by the Warsong engineers, it is now a home for the new Horde and a symbol of power and innovation. The Warchief has rewarded us generously for our work!";
 
     /// <inheritdoc/>
-    protected override string RewardDescription => $"Control of all units in Orgrimmar, {GoldReward} gold, able to train Varok and Azerite Siege Engines";
+    protected override string RewardDescription => $"Control of all units in Orgrimmar, gold, able to train Varok and Azerite Siege Engines";
```
Comment (YakaryBovine) [inline src/WarcraftLegacies.Source/Quests/Warsong/QuestOrgrimmar.cs:34]: Looks like a typo. It shouldn't mention gold at all right?

Comment (SebastianTanoni) [reply src/WarcraftLegacies.Source/Quests/Warsong/QuestOrgrimmar.cs:34]: oh yeah

**Comment by SebastianTanoni** [comment]
was that the only issue? can i merge so the plebs can play with the fixes?

**Review by YakaryBovine — APPROVED** [review]
LGTM

**Pull request merged**